examples/app.jsx

```diff
@@ -94,36 +94,29 @@ var App = React.createClass({
 
     return (
       <div>
-        <p><HoverMessage /></p>
```

Comment:
Sorry, I just read the PR description. You're right, and that is crazy. I guess this is why we use `div`s for everything.
![image](https://cloud.githubusercontent.com/assets/897931/9401404/be960696-47c6-11e5-8614-63c1f5285c7d.png)